File: 프로그래머스/unrated/120846. 합성수 찾기/합성수 찾기.py
def conf(n):
    cnt = 0
    for i in range(1,n+1):
        if n%i==0:
            cnt += 1
    return cnt

def solution(n):
    cnt = 0
    for i in range(1,n+1):
        if conf(i) >= 3:
            cnt += 1
    return cnt

File: 프로그래머스/unrated/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public boolean countNum(int n){
        int cnt = 0;
        
        for(int i=1;i<=n;i++){
            if(n%i == 0){
                cnt += 1;
            }
            if(cnt > 2) return true;
        }
        
        return false;
    }
    public int solution(int n) {
        int answer = 0;
        
        for(int i=n;i>0;i--){
            if(countNum(i)){
                answer += 1;
            }
        }
        
        return answer;
    }
}